libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -945,6 +945,7 @@ public void testGrantPrivilegeStmtBuilder() {
         printStatement("grant ALL PRIVILEGES to wolfie, anna");
         printStatement("grant ALL to wolfie, anna");
         printStatement("grant ALL to anna");
+        printStatement("grant role1, role2 to anna");
     }
```

Comment:
Could you please add same `revoke`?
`REVOKE group_role FROM role1, ... ;`

Wanted to mention deny but looks like it's not even supported in PG?

Reply:
No, deny is not supported (also for PG), I have a test to check that.